src/verify.c

```diff
@@ -641,7 +646,7 @@ int verify_main(int argc, char **argv)
 	 * FIXME: We need a command line option to override this in case the
 	 * certificate is hosed and the admin knows it and wants to recover.
 	 */
-	if (!signature_initialize(UPDATE_CA_CERTS_PATH "/" SIGNATURE_CA_CERT)) {
+	if (!initialize_signature()) {
```

Comment:
Instead of calling `initialize_signature()` for each subcommand, perhaps we should move this into `swupd_init()`?